xds/internal/balancer/edsbalancer/eds_impl_test.go

```diff
@@ -629,6 +629,51 @@ func (s) TestEDS_CircuitBreaking(t *testing.T) {
 	for _, done := range dones {
 		done()
 	}
+
+	// Send another update, with only circuit breaking update (and no picker
+	// update afterwards). Make sure the new picker uses the new configs.
+	var maxRequests2 uint32 = 10
+	edsb.updateServiceRequestsConfig("test", &maxRequests2)
+
+	// Picks with drops.
+	dones2 := []func(){}
```

Comment:
Can't we just use `dones` here as well? Do we need `dones2`?

Reply:
Done